telegram_integration.py

```diff
@@ -188,10 +190,11 @@ def feedback_handler(bot, update, args=None):
                          text="Format: /feedback [feedback]")
 
 def handle_error(bot, update, error):
-    try:
-        raise error
-    except TelegramError:
-        logging.getLogger(__name__).warning('TelegramError! %s caused by this update: %s', error, update)
+    bot.send_message(chat_id=ERROR_CHAT_ID, text=error)
```

Comment:
+ additional info, such as the last x non-bot messages and a link to the group in which the game took place?